queries.sql

```diff
@@ -8,3 +8,39 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SELECT * FROM animals;
+SAVEPOINT deleteByBirth;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO deleteByBirth;
+SELECT * FROM animals;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+SELECT * FROM animals;
+COMMIT;
+
+SELECT COUNT(name) FROM animals;
+SELECT COUNT(name) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+SELECT MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
```

Comment:
- [ ] Nice job :clap: on getting the MIN and MAX weight however, you forgot to select the species column. Here is the deference :point_down: 

#### your  output commend 
![image](https://user-images.githubusercontent.com/65068771/159663484-ae4ae7f6-0a93-4c84-b501-7af8b52d6709.png)

#### the output selected with species
![image](https://user-images.githubusercontent.com/65068771/159663746-b63ea27c-380b-4a96-b877-63ed9c1ebd86.png)

Reply:
Thank you for providing this clue, it was super useful!